Mac-App/Backend/Backend/Parsers/Yaml/YamlParser.swift

```diff
@@ -9,6 +9,12 @@
 import Foundation
 import Yams
 
+typealias YamlParserType = ([URL]) -> [SwiftDeps]
```

Comment:
Is this a common practice to add Type suffix at the end of typealiases, if not maybe we could come up with a better name? @Thurman1776-

Reply:
I am not sure if this is a common practice for typealiases but I know it is common for protocols that retrofit an existing type to allow testability (so you pass a protocol instead of the concrete type) as this cases do not fall into any of the recommendation for naming protocols from Swift API guidelines. 

I did not gave this too much thought honestly, I pretty much defaulted to the suffix `type` I use already for protocols. 

I'm not against changing it though. Do you have any suggestions?